src/Table/Table.tsx

```diff
@@ -105,13 +105,20 @@ const Table: React.FC<TableProps> = ({ className,
                     {headerGroups.map((headerGroup: any) => (
                         <tr {...headerGroup.getHeaderGroupProps()}>
                             {headerGroup.headers.map((column: any) => (
-                                <th {...column.getHeaderProps(column.getSortByToggleProps())}>
-                                    {column.render('Header')}
-                                    {column.isSorted && (
-                                        <>{(column.isSortedDesc ? <ArrowDown /> : <ArrowUp />)}</>
-                                    )}
-                                </th>
-                            ))}
+                                    <th {
+                                        ...column.getHeaderProps( column.getSortByToggleProps(
+                                            { title: 'Hold shift & click the column to add to multi-sort', width:column.width },
+                                            ))
```

Comment:
Thinking more about this, I think we need to determine if the `disableMultiSort` option is passed to determine which tooltip message we should show --- i.e., if it is `true`, we should show the normal props.title... but it if is `false`, we should show the message above.